osmosis-1/osmosis-1.assetlist.json

```diff
@@ -17192,6 +17192,47 @@
       "logo_URIs": {
         "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/osmosis/images/COCA.png"
       }
+    },
+    {
```

Comment:
Please do not add to the assetlist--just the zone_Assets file is sufficient. This file will be deleted.

Reply:
Alright...